platform/kube/inject/inject.go

```diff
@@ -460,6 +481,17 @@ func intoObject(c *Config, in interface{}) (interface{}, error) {
 		}
 	}
 
+	// skip customized exclude namespaces
+	for _, excludeNamespace := range c.ExcludeNamespaces {
+		if excludeNamespace == "" {
```

Comment:
s/`""`/`v1.NamespaceAll`

Reply:
I was removing the check here as we already guard that the excludeNamsepace cannot be allNamespaces when get initialize.